zephyr/src/utils.c

```diff
@@ -1344,4 +1344,3 @@ int insert_wpa_network_config(char *config) {
 
 void remove_pac_file(char *path) {
 }
-
```

Comment:
should be a fixup of original `no up` commit.

Reply:
> should be a fixup of original `no up` commit.
As the original` no up` commit was merged, not using fixup sauce tag for this commit.